pkgs/top-level/all-packages.nix

```diff
@@ -13724,6 +13716,20 @@ in
 
   lyx = callPackage ../applications/misc/lyx { };
 
+
+  magnetophonDSP = {
```

Comment:
Also, if you add such a package set, explain in a line or two what it is for, because it definitely isn't as obvious as say `pythonPackages` or `haskellPackages`.